client/components/app.js

```diff
@@ -10,17 +24,25 @@ import Meals from './meals.js';
 import Login from './login.js';
 import RecipeDetails from './recipe-details.js';
 
+
+
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {
-      drawerOpen: false,
-      isLoggedIn: true
-    };
 
     this.toggleDrawer = this.toggleDrawer.bind(this);
     this.renderApp = this.renderApp.bind(this);
     this.renderLogin = this.renderLogin.bind(this);
+    this.auth = new Auth('sNfZXyIkcjg3QZve68HJXoGfzFVZgjE4', 'ifeedme.auth0.com', () => {
```

Comment:
should we be calling the key obj here instead of feeding it in directly?

Reply:
Oh I forgot to reference them from client/Auth/Auth_keys file